src/app/components/content/IsaacCodeSnippet.tsx

```diff
@@ -56,7 +56,8 @@ const IsaacCodeSnippet = ({doc}: IsaacCodeProps) => {
         </div>
         {doc.url && <Row>
             <Col className="text-center mb-2">
-                <a href={doc.url} onClick={logViewOnGitHub} target="_blank" rel="noopener noreferrer">View on GitHub</a>
+                <a className="no-print" href={doc.url} onClick={logViewOnGitHub} target="_blank" rel="noopener noreferrer">View on GitHub</a>
+                <a className="only-print mb-2" href={doc.url} target="_blank" rel="noopener noreferrer">{doc.url}</a>
```

Comment:
I don't think the `mb-2` here does anything - removing it doesn't seem to affect the spacing.
```suggestion
                <a className="only-print" href={doc.url} target="_blank" rel="noopener noreferrer">{doc.url}</a>
```